nengo/neurons.py

```diff
@@ -225,12 +225,16 @@ class LIF(LIFRate):
 
     probeable = ['spikes', 'voltage', 'refractory_time']
 
+    def __init__(self, min_voltage=0, **kwargs):
```

Comment:
We shouldn't change the order of parameters. And in general, I don't think we should use `**kwargs` if we can avoid it (and we can in this case)